static/src/js/util/url/__tests__/projectCollections.test.js

```diff
@@ -298,6 +301,103 @@ describe('url#decodeUrlParams', () => {
       expect(decodeUrlParams('?p=!collectionId1!collectionId2&pg[1][m]=harmony')).toEqual(expectedResult)
     })
   })
+  describe('enable spatial subsetting flag', () => {
```

Comment:
you didn't add `test('decodes enable spatial subsetting correctly when not encoded', () => {`